sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Get-SPOTheme.md

```diff
@@ -24,6 +24,11 @@ Get-SPOTheme [[-Name] <String>] [<CommonParameters>]
 
 The **Get-SPOTheme** cmdlet returns the settings for a named existing theme, or for all uploaded themes if no name is provided.
 
+> [!NOTE]
+> To support consistent branding and simplify governance, theme management is transitioning to a centralized model.
+> - The **primary geo** will act as the central location for **organization-wide theme creation and management**. Themes created here will be visible and applicable across satellite geos.
+> - The **Satellite Geo Administrators** will be able to view and apply themes from the primary geo to their own sites by using `Get-SPOTheme` and `Set-SPOWebTheme` cmdlet. Themes previously created within satellite geos will remain available for use. However, **creating new themes in satellite geos will no longer be supported** going forward.
```

Comment:
>The **Satellite [](http://example.com/codeflow?start=4&length=15)

```
The satellite
```

Reply:
(move "satellite" outside of the **, since there's no concept of "Satellite Geo Administrator", which is how this looks like when all that is bolded)

Reply:
updated